bin/cake.php

```diff
@@ -16,17 +16,8 @@
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
 
-$minVersion = '5.6.0';
-if (file_exists('composer.json')) {
```

Comment:
While this was a nice way of checking, it is also a slow way (file_exists check, file_get_contents, json_decode, preg_replace)